### ISSUE TITLE
No module named 'scan_notes'

### ISSUE THREAD

**Issue by Sitin** [adamyi/notes_to_keep#1]
OS X 10.13.6, Python 3.7 (Homebrew), fresh virtual environment

```
$ notes_to_keep <email> <pass>
Traceback (most recent call last):
  File "/Users/sitin/.virtualenv/1way/bin/notes_to_keep", line 7, in <module>
    from notes_to_keep.notes_to_keep import main
  File "/Users/sitin/.virtualenv/1way/lib/python3.7/site-packages/notes_to_keep/notes_to_keep.py", line 29, in <module>
    from scan_notes import ScanNotes
ModuleNotFoundError: No module named 'scan_notes'
```

**Comment by Sitin** [comment]
Ah, I see, it's just not Python 3 compatible...

**Comment by Faycop** [comment]
Did you manage to resolve this issue in the end? How?

**Comment by ghrene** [comment]
@Sitin How could I overcome that issue, have you got an idea? Or do you rather see another way of copying notes? Thx a lot in advance!

**Comment by adamyi** [comment]
This should be solved with version 0.0.2. Please use `--no-label` if using Python 3.7.

Let me know if there's any more issue with Python 3.
